data/provider/src/main/java/tech/pegasys/artemis/api/ChainDataProvider.java

```diff
@@ -23,6 +23,7 @@
 import tech.pegasys.artemis.api.schema.BeaconHead;
 import tech.pegasys.artemis.api.schema.Committee;
 import tech.pegasys.artemis.api.schema.SignedBeaconBlock;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
```

Comment:
note: i'll have to fix this when i do beaconState - cant return this internal structure... for now there's nothing that can be done about it.